runtime/compiler/z/codegen/J9TreeEvaluator.cpp

```diff
@@ -3758,8 +3758,12 @@ J9::Z::TreeEvaluator::ArrayCopyBNDCHKEvaluator(TR::Node * node, TR::CodeGenerato
    return NULL;
    }
 
+/*
+ * Generate instructions to fill in the J9JITWatchedStaticFieldData.fieldAddress, J9JITWatchedStaticFieldData.fieldClass for static fields,
+ * and J9JITWatchedInstanceFieldData.offset for instance fields at runtime. Used for fieldwatch support.
+*/
```

Comment:
Aren't these comments redundant? 

This function has already been doxygen documented inside `J9TreeEvalautor.hpp`:
https://github.com/eclipse/openj9/blob/5242f38bdf9456861bb342c6bfab959dbc8571d3/runtime/compiler/z/codegen/J9TreeEvaluator.hpp#L463-L466

The same comment should be available on X and Z.